### ISSUE TITLE
Get total training time from Azure experiment.

### ISSUE THREAD

**Issue by zewditu** [dotnet/machinelearning-modelbuilder#514]
**Model Builder Version**: _In which Model Builder version was the bug encountered?_
**Visual Studion Version**: _In which Visual Studio version was the bug encountered?_

**Bug description**

The current training-time in evaluate page and duration in explored model table are not coming  from azure rather  get from Model Builder from start and stop training time elapsed. This time is not seems the actual training time.

**Steps to Reproduce**
1. _List the minimal steps required to reproduce the bug._
2. _Thanks for reporting!_

**Expected Experience**
_A description of what you expected to happen. If applicable, add screenshots or "Machine Learning" Output Logs to help explain what you expected._

**Actual Experience**
_A description of what actually happens. If applicable, add screenshots or "Machine Learning" Output Logs to help explain what actually happened._

**Additional Context**
_Add any other context about the problem here, such as unusual system configurations, network restrictions, etc._

**Comment by beccamc** [comment]
Per discussion with Bri, Azure training time should include the whole time user is running the experiment, even if Azure isn't actually training yet (preparing compute, etc. included). I did not update to use the training time by azure. Instead I renamed the Azure Training time to Total time. Local experiences should still say TrainingTime

![image.png](https://images.zenhubusercontent.com/5d55c3d68fac840001314eaa/c960d91c-a428-4c15-af2b-9daaaaaeceb9)
